apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -519,6 +605,39 @@ const EventTypePage = (props: EventTypeSetupProps) => {
           }}
         />
       ) : null}
+
+      <Dialog open={unsavedDialogOpen} onOpenChange={setUnsavedDialogOpen}>
+        <ConfirmationDialogContent
+          isLoading={updateMutation.isLoading}
+          variety="danger"
+          cancelBtnText="Discard Changes"
+          // confirmBtnText="Keep Editing"
+          title={t(`delete${"_managed"}_event_type`)}
```

Comment:
Need to add the translation string to L615: place the string in common.json then use the key here

Reply:
@sean-brydon, which common.json? there are many.

Reply:
"delete_managed_event_type" already exists in common.json.

@Krishna2323 if you have to add any new translation add it in locales/en/common.json.  you can also check i18next docs